caffe2/mobile/contrib/ios/mpscnn/mpscnn.mm

```diff
@@ -257,11 +257,10 @@ void computeOutputHW(
     int* OH,
     int* OW) {
   Tensor input = caffe2::empty({1, 1, H, W}, at::dtype<float>().device(CPU));
-  Tensor output(CPU);
```

Comment:
Wow, the old version of this code is really wacky. I guess it just drops `output` on the floor when it's done, previously?

Reply:
Yes that is correct.